api_app/helpers.py

```diff
@@ -130,3 +133,43 @@ def get_hash_type(hash_value):
             detected_hash_type = hash_type
             break
     return detected_hash_type  # stays None if no matches
+
+
+def get_custom_config_as_dict(user, entity_type, name=None) -> dict:
```

Comment:
I think this should be better placed as a `classmethod` in the corresponding CustomConfig class. In that way we don't separate the logic that leverages that class from the class itself. This is very important for code maintainability, in particular when the project grows up

Reply:
Done